cfgov/unprocessed/js/routes/on-demand/youth-employment-programs/buying-a-car/index.js

```diff
@@ -27,9 +27,7 @@ const expandableEls = Array.prototype.slice.call(
 
 const expandableData = expandableEls.reduce( ( memo, expandable ) => {
   const labelText = expandable.querySelector( `${ EXPANDABLE_UI_CLASSES.label }` );
-  const content = expandable.querySelector( `
-    ${ EXPANDABLE_UI_CLASSES.content } .rich-text
-  ` );
+  const content = expandable.querySelector( `${ EXPANDABLE_UI_CLASSES.content }` );
```

Comment:
This is for this page https://www.consumerfinance.gov/practitioner-resources/resources-youth-employment-programs/buying-a-car/